PO.md

```diff
@@ -0,0 +1,54 @@
+4. Czytelność backlogu i Kryteria Akceptacji User Stories dla zespołu DEV
+5. Pilnowanie budżetu projektu i ROI z inwestycji
+6. Zdany​ ​scrum.org​ Product Owner Open Assessment (junior) lub uzyskanie
+certyfikatu Proffesional Scrum Product Owner (mid / senior) przed rozpoczęciem projektu.
+
+### Główne Narzędzie pracy PO
+1. Jira
+2. Planner
+3. Cost Tracker
+   
+### Obowiązki Product Ownera
+* Zarządzanie Backlogiem Produktu
+* Opracowywanie i komunikacje Celu Produktu
+* Negocjacje z zespołem zakresy sprintów i odbiera ich realizację;
+* Publiczne opowiadanie o projekcie / produkcie, występowanie w materiałach
+promocyjnych
+* Maksymalizacja wartości pracy będącej efektem działań Scrum Teamu;
+* Monitorowanie budżetu i końcowy wynik produktu;
+* Planowanie Kamieni Milowych oraz ich realizacje wedle ustalonych z
+Klientem terminów;
+* Odpowiada za jasne zakomunikowanie celu projektu oraz elementów
+Backlogu Projektu Zespołowi Scrumowemu;
+* Odpowiada za komunikację z Interesariuszami i sponsorami w obszarze
+postępu prac oraz timeline wydań
+* Zarządza ryzykami oraz ich mitygacją na poziomie produktu;
+* Tworzy Elementy Backlogu Produktu i Zarządza Backlogiem Produktu przez :
+* Określanie potrzeb biznesowych w formie User Story zgodnie z modelem ​I.N.V.E.S.T​ w oparciu o całą posiadaną wiedzę na temat produktu, który powinien być aktualizowany wraz ze pojawiającymi się nowymi potrzebami biznesowymi;
+* Opracowywanie propozycji kryterium akceptacji dla każdego User Story w Backlogu Produktu, które może być doprecyzowywane w trakcie Refinementu lub Planowania;
+* Odpowiada za priorytetyzację Backlogu Produktu;
+* Jest odpowiedzialny za zbieranie i filtrowanie potrzeb interesariuszy
+oraz decyzję o ich uwzględnieniu w Backlogu w celu maksymalizacji Wartości Produktu.
+Obowiązki w obszarze Kontrolingu
+* Planowanie, aktualizowanie zapotrzebowania na członków zespołu w Tempo Planner;
+* Analizę i planowanie zespołu Tempo Planner w oparciu o informacje dostępne w Cost Tracker
+  design by​ Laboratorium EE
+* Przygotowywania na koniec Sprintu raportu dotyczącego stopnia realizacji budżetu z perspektywy ostatniego wydanego Inkrementu;
+Obowiązki Dotyczące Narzędzi
+* Aktualizowanie na bieżąco Backlogu Produktu w Jirze;
+* Aktualizowanie na bieżąco Roadmapy Produktu;
+
+
+Poduct Owner może wykonywać powyższe zadania samodzielnie lub zlecać je innym osobom. Niemniej to Product Owner pozostaje za nie odpowiedzialny. Aby praca Product Ownera mogła przynosić pożądane efekty, jego decyzje muszą być respektowane przez całą organizację.
+Decyzje te uwidoczniają się w treści oraz kolejności elementów Product Backlogu oraz w Inkremencie, który może zostać poddany ocenie podczas Sprint Review. Product Owner to jedna osoba, nie komitet. Product Owner może w Product Backlogu uwzględniać potrzeby licznych interesariuszy. Ci, którzy chcą dokonać zmian w Product Backlogu mogą to zrobić, próbując przekonać do tego Product Ownera.
```

Comment:
Tak jak wyżej - jeśli te akapity mają się wyświetlać osobno to potrzebna jest między nimi linia przerwy